### DISCUSSION TITLE
Create new Module Template

### DISCUSSION THREAD

**Discussion by rchibnik** [oqtane/oqtane.framework#1590] [Q&A] [answered]
Can someone point me to the module template location so that I can create a custom template

I'd like to create a module template and include the necessary references to the Radzen components.

Thanks

**Comment by horacioj** [comment]
It could be this? https://github.com/oqtane/oqtane.framework/blob/dev/Oqtane.Client/Modules/Admin/ModuleCreator/Index.razor

**Comment by leigh-pointer** [comment, marked as answer]
Have a look here
Oqtane.Server/wwwroot/Modules/Templates/External
________________________________

**Reply by rchibnik** [reply]
Thanks. Looked a lot of places - but not there.

**Reply by rchibnik** [reply]
As a follow up, in case anyone else is interested - I copied the template "External" folder and its contents to a new sibling folder and customized the classes and configurations as outlined in https://blazorhelpwebsite.com/ViewBlogPost/46 and was able to create modules with working Radzen components. 

The only thing I couldnt figure out was how to add the Radzen.Blazor package to the new template, so after creating the new module you have to add the package with nuget.

Thanks again for the pointer.
